proxy/envoy/resources.go

```diff
@@ -175,12 +186,17 @@ type HTTPRoute struct {
 	faults []*HTTPFilter
 }
 
+// CatchAll returns true if the route matches all requests
+func (route *HTTPRoute) CatchAll() bool {
+	return len(route.Headers) == 0 && route.Path == "" && route.Prefix == "/"
+}
```

Comment:
if my http route rule does not provide path/prefix/headers, will route.Prefix still be set to / ?